tacc_stats/analysis/gen/jid_table.py

```diff
@@ -1,13 +1,16 @@
-import time
+import os,sys,time
+sys.path.append("/home/sg99/tacc_stats")
 import psycopg2
-import tacc_stats.cfg as cfg
+import conf_parser as cfg
```

Comment:
Conf parser needs to be a full name under taccstats like the line below: from tacc_stats.analysis.gen.utils import read_sql

Reply:
This is a comment for every file